### ISSUE TITLE
Compatibility with Lighthouse 10.0.0

### ISSUE THREAD

**Issue by pwppg** [googleads/publisher-ads-lighthouse-plugin#387]
New version of Lighthouse is available - https://github.com/GoogleChrome/lighthouse/releases/tag/v10.0.0

Unfortunately current publisher ads plugin is not compatible with this version of Lighthouse.

`
Runtime error encountered: Cannot find module 'lighthouse/lighthouse-core/lib/i18n/i18n.js'
Require stack:
- /usr/lib/node_modules/lighthouse-plugin-publisher-ads/plugin.js
Error: Cannot find module 'lighthouse/lighthouse-core/lib/i18n/i18n.js'
Require stack:
- /usr/lib/node_modules/lighthouse-plugin-publisher-ads/plugin.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:924:15)
    at Function.Module._load (node:internal/modules/cjs/loader:769:27)
    at Module.require (node:internal/modules/cjs/loader:996:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/usr/lib/node_modules/lighthouse-plugin-publisher-ads/plugin.js:15:14)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:201:29)
`

**Comment by connorjclark** [comment]
`1.5.7-beta` may work in the meantime.

**Comment by pwppg** [comment]
True. I've tested **1.5.7-beta** with Lighthouse **10.0.1** and it works.

**Comment by connorjclark** [comment]
after #395, could you publish a new version of pubads @jburger424 ?
